backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -103,7 +114,9 @@ describe('CommunityResolver', () => {
               {
                 id: 2,
                 foreign: homeCom2.foreign,
-                publicKey: expect.stringMatching('publicKey-HomeCommunity'),
+                publicKey: expect.stringMatching(
+                  '5b47388f9e8db5416201e485398ed0d72ab20d9ee951ccc1754245278e3ae6c6',
+                ),
```

Comment:
Why not using the variable `homeCom2.publicKey` instead of hardcoded string?

Reply:
Community.publicKey should be a Buffer type, graphql returns a hex string.
But you right I could make it way better. Save for future test, if the key will be actually validated.